actions/dependabot-gar-login/README.md

```diff
@@ -0,0 +1,39 @@
+# Dependabot GAR Login
+
+Updates the token for Google Artifact Registry and sets it as a secret for Dependabot at the repository level.
+
+The token can be refreshed and set to the `DEPENDABOT_GAR_TOKEN` secret every 50 minutes to ensure Dependabot can access Google Artifact Registry without interruptions.
+
+## Example
+
+```yaml
+name: Update Google Artifact Registry Token
+
+on:
+  schedule:
+    # Update every 50 minutes
+    - cron: "*/50 * * * *"
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
```

Comment:
Do you need content-read if the action does not come from a relative path? I'm also wondering, don't you need some kind of write-permissions to actually modify a secret?

Reply:
Ah, makes sense. Thank you :)